build.gradle

```diff
@@ -281,3 +281,107 @@ release {
     requireBranch = 'release-.*|master'
   }
 }
+
+project.ext.allProjectNames = [
```

Comment:
I overlooked that property when I was originally looking around, thanks for pointing it out! That simplifies things a lot. I actually used a related one instead, `rootProject.subprojects` (because I want to omit the root project from the list).